### PR TITLE
libgxml: updated to 0.16.2 (moved to meson).

### PR DESCRIPTION
**Pull request by KrullBorg**

### CODE REVIEW

**Comment by lazka** [comment]
```
Meson encountered an error in file gxml/meson.build, line 166, column 0:
Program "g-ir-compiler" not found or not executable
```

**Comment by KrullBorg** [comment]
could it be merged?

File: mingw-w64-gxml/PKGBUILD [hunk 1]
```diff
@@ -14,44 +14,43 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
...
+sha256sums=('e2c68c2a59c351066469cbf93ccae51c796f49e0d48639aed72af39c544dcef3'
+            'c2520ba69518d0979f0168ecda0bc827424adce0fab6c967a6717e97e9677fbe')
 
 prepare() {
   cd "${srcdir}"/${_realname}-${pkgver}
-  patch -p1 -i ${srcdir}/001-fix-attributes-traversing.patch
 
-  # Get proper unprefixed directories for gobject-introspection files
-  autoreconf -fiv
+  echo "@VALADEPS" > gxml/gxml.deps.in
```
Comment (lazka) [inline mingw-w64-gxml/PKGBUILD]: "@VALADEPS@" ?

Comment (KrullBorg) [reply mingw-w64-gxml/PKGBUILD]: i don't know: it's a file missing in tarball

Comment (lazka) [reply mingw-w64-gxml/PKGBUILD]: Yeah, but I think you missed the trailing "@"

Comment (lazka) [reply mingw-w64-gxml/PKGBUILD]: (I've filed https://gitlab.gnome.org/GNOME/gxml/issues/2)

Comment (KrullBorg) [reply mingw-w64-gxml/PKGBUILD]: i tried to register to gitlab but i got always error 500; so i had report some problems via mail to developers, but they didn't reply yet

**Pull request merged**